config.js

```diff
@@ -10,6 +10,7 @@ const config = {
     appId: 'rwallet',
     javascriptKey: '',
     serverURL: PARSE_SERVER_URL,
+    rwalletApiKey: RWALLET_API_KEY,
```

Comment:
the header key should be "rwallet-api-key", where is that defined?

double check the key with @huhu2356